### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by amita-roy**
We have 3 classes (App, Project, and Todo) in our project. 
We have created a test folder in src directory.
The test folder contains a test file for all three classes
You can run all the test cases with  ```npm run watch```

### CODE REVIEW

File: src/test/app.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import App from '../components/app';
+import Todo from '../components/todo';
+
+const app = new App();
```
Comment (bellom) [inline src/test/app.test.js:4]: You have really done well, great job guys.
Perhaps due to workload 🙂 , you missed testing the method `addTodo` in `app.js`
kindly add it.

I trust you guys can add this before merging 👀  on you.

**Review by bellom — APPROVED** [review]
## Hi Team :clap: 

##### You have done wonderfully well, you passed all checklist for this project :+1:
 
**Kindly update the `app.test` file before merging to `dev`**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck with your next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun J. Bello**
**_Slack Handle - @sean bellom_**

File: src/test/app.test.js (hunk 1 above)
Comment (amita-roy) [reply src/test/app.test.js:4]: Thanks for the compliment and we assure you that we will first add that test and then we will merge the branch.

**Comment by amita-roy** [comment]
Added the test for adding new todo in the project and now merging this branch to  the development branch

Screenshot 2021-02-22 at 8 43 01 AM

**Pull request merged**